### PR TITLE
chore: remove unneeded @cloudflare/ai dependency

### PR DESCRIPTION
**Pull request by Cherry**
### Details

Cloudflare ships the `Ai` type directly in `workers-types` now so there's no need to add `@cloudflare/ai` as a separate dependency.

This bumps `workers-types`, and then removes the no longer needed `@cloudflare/ai` package.

### CODE REVIEW

File: packages/experimental/labs-workers/package.json [hunk 1]
```diff
@@ -30,7 +29,7 @@
     "modelfusion": "^0.137.0"
   },
   "devDependencies": {
-    "@cloudflare/workers-types": "^4.20240423.0",
+    "@cloudflare/workers-types": "^4.20240502.0",
```
Comment (richburdon) [inline packages/experimental/labs-workers/package.json:32]: we'd need to bump packages/apps/composer/app also to be in sync.

**Review by richburdon — APPROVED** [review]
thanks for spotting this.

File: packages/experimental/labs-workers/package.json (hunk 1 above)
Comment (Cherry) [reply packages/experimental/labs-workers/package.json:32]: good catch, updated this now!

**Comment by Cherry** [comment]
No longer needed after https://github.com/dxos/dxos/pull/6448 was merged

**Pull request closed without merging**